docs/design.md

```diff
@@ -35,15 +35,22 @@ MicroShift aims at meeting all of the following design goals:
   * MicroShift makes frugal use of system resources. It runs on <1GB RAM and <1 CPU core (Intel Atom- or ARM Cortex-class). It consumes <500MB on the wire (per install/update) and <1GB at rest (excl. etcd state).
 
 * **Consistency with OpenShift:**
-  * MicroShift runs all workloads that OpenShift runs, except those which depend on OpenShift's cluster operators.
-  * MicroShift clusters can be managed like OpenShift clusters through [Open Cluster Management](https://github.com/open-cluster-management), except where functions depend on OpenShift's cluster operators.
+  * MicroShift is binary compatible with OpenShift and Kubernetes conforming.
+  * MicroShift runs all OpenShift workloads without modification, except for if these rely on OpenShift APIs that are
+    * provided by OpenShift's Operators for cluster infrastructure or lifecycle management (not applicable in MicroShift and thus removed) or
+    * not relevant to a pure non-interactive, runtime-only cluster (as opposed to a build-cluster with multiple interactive users).
+  * MicroShift workloads can be managed like OpenShift workloads through [Red Hat Advanced Cluster Management for Kubernetes](https://www.redhat.com/en/technologies/management/advanced-cluster-management), except where functions depend on unsupported OpenShift APIs (see above).
 
 
 ## Design Principles
 When deciding between different design options, we follow the following principles:
 
 * **Minimal core**: We keep MicroShift to a minimal set of functionality, but provide mechanisms for extension.
-  * Discriminator: If a functionality can be added post-cluster-up with reasonable effort, then it should not be part of the MicroShift core/binary.
+  * Discriminator 1: If a functionality implements changes to the OS or its configuration, it should probably be done as pre-requisite for running MicroShift binary.
+  * Discriminator 2: If a functionality is essential to MicroShift's atomic start/stop/update behavior (i.e. MicroShift running/stopped means the OpenShift control plane is running/stopped) it must be part of the binary, otherwise it should be hosted on the cluster.
+  * Discriminator 3: If can be installed "post-cluster-up" and isn't used by 90% of MicroShift users, it should probably not be pre-integrated with MicroShift at all.
+* **Least-privileged**: We minimize privileges needed by MicroShift and its workloads; unlike with OpenShift, even kubeadmin should not be able to compromise the host system.
```

Comment:
this claim seems too strong.  kubeadmin can still deploy workloads that run root.

Reply:
Agreed. Removed that.